tests/test_hazenlib.py

```diff
@@ -1,89 +1,207 @@
+print(TEST_DICOM.Columns * TEST_DICOM.PixelSpacing[0])
 
 class TestHazenlib(unittest.TestCase):
-    # Data by ImplementationVersionName
-    # all test values are taken from DICOM headers
 
-    ROWS = 512
-    COLUMNS = 512
-    TR_CHECK = 500
-    BW = 205.0
+
+    def test_intit(self):
+        test_dicoms = {'philips': {'file': str(TEST_DATA_DIR / 'resolution' / 'philips' / 'IM-0004-0002.dcm'),
+                                   'MANUFACTURER': 'philips',
+                                   'ROWS': 512,
+                                   'COLUMNS': 512,
+                                   'TR_CHECK': 500,
+                                   'BW': 205.0,
+                                   'ENHANCED': False,
+                                   'PIX_ARRAY': 1,
+                                   'SLICE_THICKNESS':5,
+                                   'PIX_SIZE': [0.48828125, 0.48828125],
+                                   'AVERAGE': 1},
+                        'siemens': {'file': str(TEST_DATA_DIR / 'resolution' / 'eastkent' / '256_sag.IMA'),
+                                    'MANUFACTURER': 'siemens',
+                                    'ROWS' : 256,
+                                    'COLUMNS' : 256,
+                                    'TR_CHECK' :  500,
+                                    'BW' : 130.0,
+                                    'ENHANCED': False,
+                                    'PIX_ARRAY': 1,
+                                    'SLICE_THICKNESS': 5,
+                                    'PIX_SIZE':  [0.9765625, 0.9765625],
+                                     'AVERAGE': 1},
+                        'toshiba': {'file': str(TEST_DATA_DIR / 'toshiba' / 'TOSHIBA_TM_MR_DCM_V3_0.dcm'),
+                                    'MANUFACTURER': 'toshiba',
+                                    'ROWS': 256,
+                                    'COLUMNS': 256,
+                                    'TR_CHECK': 45.0,
+                                    'BW': 244.0,
+                                    'ENHANCED': False,
+                                    'PIX_ARRAY': 1,
+                                    'SLICE_THICKNESS': 6,
+                                    'PIX_SIZE':  [1.0, 1.0],
+                                    'AVERAGE': 1},
+                        'ge': {'file': str(TEST_DATA_DIR / 'ge' / 'ge_eFilm.dcm'),
+                                    'MANUFACTURER': 'ge',
+                                    'ROWS': 256,
+                                    'COLUMNS': 256,
+                                    'TR_CHECK': 1000.0,
+                                    'BW': 156.25,
+                                    'ENHANCED': False,
+                                    'PIX_ARRAY': 1,
+                                    'SLICE_THICKNESS': 5,
+                                    'PIX_SIZE':  [0.625, 0.625],
+                                    'AVERAGE': 1}}
+
+
+        for manufacturer in test_dicoms.keys():
```

Comment:
this is amazing thank you! The only thing I would suggest is that we keep each test as a separate function  e.g.:

```
def test_get_manufacturer(self):
    for manufacturer in test_dicoms.keys():
        with pydicom.read_file(test_dicoms[manufacturer]['file']) as dcm:
            assert hazenlib.get_manufacturer(dcm) == test_dicoms[manufacturer]['MANUFACTURER']

def test_get_rows(self):
    for manufacturer in test_dicoms.keys():
        with pydicom ... etc
```

This means that when a test fails we can see which function is causing the error. As this is currently we have it all in one function, which means execution will stop if one error is raised and it will be harder to work out the origin of the error.